### DISCUSSION TITLE
Auto Screen-switch / Mouse follow

### DISCUSSION THREAD

**Discussion by Benno1308** [rustdesk/rustdesk#7437] [Feature Request]
### Short Summary
I would like to request a feature that allows the client to automatically follow the remote mouse cursor.

### Scenario
1. I'm connected to someone via RustDesk
2. The remote side has multiple monitors connected
3. The remote side is actively using his mouse across all monitors

### Expected Behaviour
Whenever the remote side moves the mouse cursor to a new monitor, 
I expect my RustDesk client to automatically follow the cursor to that monitor.

This can also be extended to be based on which window is currently focused, 
allowing to follow the remote work based on either where the mouse is, or which window is currently active.

### Bounty
I am willing to add a bounty of 750 USD via [polar.sh](https://polar.sh/rustdesk/rustdesk), or a similar service, if there is a reasonable timeline to add this feature

### Reference
#2059 (Creating a seperate feature request as this one adds the request for follow window focus)

https://support.anydesk.com/knowledge/display
```
If the remote device has multiple monitors, enabling "Follow remote cursor" automatically switches the AnyDesk view to the correct monitor if the remote user moves the cursor to a different monitor.
If the remote device has multiple monitors, enabling "Follow remote window focus" automatically switches the AnyDesk view to the display where the foreground window is located.
```

**Comment by beelchester** [comment]
Hello 👋🏻
I'm interested in working on this feature. I will start my work today :)

**Comment by rustdesk** [comment]
@Benno1308 Please go to https://github.com/rustdesk/rustdesk/issues/7439 for funding it if you are ok with @sahilyeole's timeline.

**Reply by Benno1308** [reply]
Done

**Comment by rustdesk** [comment]
Finished, https://twitter.com/rustdesk/status/1783468475480191022
